src/index.ts

```diff
@@ -29,7 +29,7 @@ Sentry.init( {
 	dsn: 'https://97693275b2716fb95048c6d12f4318cf@o248881.ingest.sentry.io/4506612776501248',
 	debug: true,
 	enabled: process.env.NODE_ENV !== 'development',
-	release: COMMIT_HASH,
+	release: app.getVersion() ? app.getVersion() : COMMIT_HASH,
```

Comment:
[app.getVersion()](https://www.electronjs.org/docs/latest/api/app#appgetversion) is an Electron method that returns the version of the loaded application found in the app package.json file.

Reply:
I paused to think if we should/need to update/remove the existing `COMMIT_HASH` references here (or elsewhere). My thought is that retaining the references still makes sense.

Many of the other references appear to log _both_ values, so having both likely is intentional and helpful context. I'm not sure under what circumstances this particular line would use the `COMMIT_HASH`.